### ISSUE TITLE
Have not added the contract itself

### ISSUE THREAD

**Issue by olegabr** [RyuuGan/sol-merger#21]
First, the recommended `npm run build-contracts` command has failed:

```
$ npm run build-contracts

> ERC777iHomeCredits@1.0.0 build-contracts /home/olegabr/work/ERC777iHomeCredits
> sol-merger "./contracts/*.sol" ./build

,
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! ERC777iHomeCredits@1.0.0 build-contracts: `sol-merger "./contracts/*.sol" ./build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the ERC777iHomeCredits@1.0.0 build-contracts script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/olegabr/.npm/_logs/2019-10-15T10_27_23_541Z-debug.log
```

I've worked it around with:

```
./node_modules/sol-merger/dist/bin/sol-merger.js "./contracts/*.sol" ./build
```

and it produces file in the build directory, and this file included all but the `ERC777iHomeCredits` contract definition. I've added it manually and verification succeeded.

**Comment by RyuuGan** [comment]
Hello @olegabr,

Can you send me the reproducable example of the contract that doesn't work?

**Comment by olegabr** [comment]
@RyuuGan sure!

```solidity
pragma solidity ^0.5.1;

import "openzeppelin-solidity/contracts/token/ERC777/ERC777.sol";

contract ERC777Test is ERC777(
  "Test",
  "TSX",
  new address[](0)
) {
  constructor() public {
    _mint(
      msg.sender,
      0x476Bb28Bc6D0e9De04dB5E19912C392F9a76535d,
      100000000000 * 10 ** uint256(decimals()),
      "",
      ""
    );
  }
}
```

**Comment by RyuuGan** [comment]
Should be fixed in `1.1.0`

**Comment by olegabr** [comment]
Thank you!
